mmv1/third_party/terraform/tests/resource_eventarc_google_channel_config_test.go.erb

```diff
@@ -0,0 +1,197 @@
+<% autogen_exception -%>
+package google
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"testing"
+
+	dcl "github.com/GoogleCloudPlatform/declarative-resource-client-library/dcl"
+	<% if version == "ga" -%>
+	eventarc "github.com/GoogleCloudPlatform/declarative-resource-client-library/services/google/eventarc"
+	<% else -%>
+    eventarc "github.com/GoogleCloudPlatform/declarative-resource-client-library/services/google/eventarc/beta"
```

Comment:
nit: These are spaces while other lines are tabs, would you mind converting these?

It's a template file, so editors don't work _quite_ right.

Reply:
Acknowledged

Reply:
Done